src/functions/get/domain/config.mock.ts

```diff
@@ -0,0 +1,9 @@
+export const config = {
+  googleAnalyticsID: 'dev-ga-id',
+  journal : {
+    numberOfDaysToShow: 2,
```

Comment:
This has to be at least 3 for being able to navigate through the weekend

Reply:
That is really the number of working days. Saturday and Sunday are treated in TARS as non working days. See the examples at the bottom of https://wiki.i-env.net/display/MES/Journal+look-ahead+logic for clarification

Reply:
That's fine, however from the mobile app's point of view `numberOfDaysToShow` is 4 if current date is Friday.

But this is not important at the moment, this data is not going to be used anywhere for now.

Reply:
Not if it's Easter 😂 If today is Thursday and tomorrow is Good Friday and Monday is Easter Monday, the app should display Thursday and Tuesday - that's 6 days.